data/birth-dates/07.txt

```diff
@@ -1 +1 @@
-Tim Berners-Lee: 8 June ????
```

Comment:
No, that is not the correct date. He would is older than the internet itself

Reply:
Added another commit, was typo - correct date is 1955

Reply:
Thanks